wbooks/src/redux/book/reducer.ts

```diff
@@ -0,0 +1,25 @@
+import { BOOKS } from '@constants/types';
+
+import { actionCreators } from './actions';
+
+const initialState = {
+  ok: false,
+  data: []
+};
```

Comment:
No, the initialState must be:
```
const initialState = {
 books: [],
 booksLoading: false,
 bookError: null
}
```
`ok`  and `data` are the `apisauce` or service response, not what you save in redux